src/commands/hearthstone.js

```diff
@@ -0,0 +1,38 @@
+const Aquarius = require('../aquarius');
+const hdb = require('hearthstone-db');
+
+class Hearthstone extends Aquarius.Command {
+  constructor() {
+    super();
+    this.description = 'Displays Hearthstone Card Image';
+  }
+
+  helpMessage() {
+    let msg = super.helpMessage();
+
+    msg += 'Usage:\n';
+    msg += '```[[Tirion Fordring]]```';
+    return msg;
+  }
+
+  message(msg) {
+    const hearthInput = Aquarius.Triggers.cardTrigger(msg);
+
+    if (hearthInput) {
+      hearthInput.forEach(cardInput => {
+        let cardName = `${cardInput[1]}`;
```

Comment:
I'd say change this to be

```javascript
const cardName = cardInput[1];
```

We don't change the value ever, and since it's a string already we don't need to cast it as one.